src/integrationtests/java/com/aws/greengrass/integrationtests/ipc/AuthorizeClientDeviceActionTest.java

```diff
@@ -82,17 +77,17 @@ private static void authzClientDeviceAction(GreengrassCoreIPCClient ipcClient,
     }
 
     @BeforeEach
-    void beforeEach(ExtensionContext context) throws DeviceConfigurationException {
+    void beforeEach(ExtensionContext context) {
         ignoreExceptionOfType(context, SpoolerStoreException.class);
         ignoreExceptionOfType(context, NoSuchFileException.class); // Loading CA keystore
 
         // Set this property for kernel to scan its own classpath to find plugins
         System.setProperty("aws.greengrass.scanSelfClasspath", "true");
         kernel = new Kernel();
-        kernel.getContext().put(GreengrassServiceClientFactory.class, clientFactory);
-
-        when(clientFactory.fetchGreengrassV2DataClient()).thenReturn(v2DataClient);
 
+        // Set up Iot auth client
+        IotAuthClientFake iotAuthClientFake = new IotAuthClientFake();
+        kernel.getContext().put(IotAuthClient.class, iotAuthClientFake);
```

Comment:
TODO: these are not even close to being actual integration tests. We should remove the rest of these mocks, but will do that in a different PR (we have separate tasks for these integ tests)